src/components/homepage/Homepage.js

```diff
@@ -0,0 +1,15 @@
+import Container from '@mui/material/Container';
+import Countries from './Country';
+import './Homepage.css';
+import Banner from './Banner';
+import Navbar from '../container/navbar/Navbar';
+
+const Homepage = () => (
+  <Container className="main-container">
+    <Navbar />
+    <Banner />
+    <Countries />
+  </Container>
```

Comment:
- [ ] **[OPTIONAL]** Currently, the home page is not displaying any countries again after you made some changes. Please for you to have a working and professional app, try working on it so that the home page displays all countries as required.

Before changes | After changes
:---:|:---:
![](https://i.imgur.com/91pVK3m.png) | ![](https://i.imgur.com/aRhbgyu.png)

Reply:
@julie-ify Thank you very much, I'll have that sorted